### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by shekhar108**
_Please delete all sections in italics before submitting this PR._

**Checks**

- [x] I've checked there are no linting errors.
- [ ] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

**Are you happy to be listed as a contributor on [Shortcuts.fun](https://shortcuts.fun)?**

_Yes_ 

**Any other information / comments**

### CODE REVIEW

File: __tests__/actions/URL.spec.ts [hunk 1]
```diff
@@ -31,4 +42,20 @@ describe('URL function', () => {
     expect(actual).toEqual(expected);
   });
 
+  it('inverts a URL action', () => {
+    if (URL.invert) {
+      const action = {
+        WFWorkflowActionIdentifier: 'is.workflow.actions.url' as WFWorkflowActionIdentifier,
+        WFWorkflowActionParameters: {
+          WFURLActionURL: '' as WFWorkflowActionParameters['WFURLActionURL'],
```
Comment (joshfarrant) [inline __tests__/actions/URL.spec.ts]: This line doesn't need the `as WFWorkflowActionParameters['WFURLActionURL']`. There's an argument for keeping it, but I'd prefer to get rid of it for now as `WFURLActionURL` is just a string anyway.

This means that this line will simply become:
```js
WFURLActionURL: '',
```

You'll then be able to remove the `WFWorkflowActionParameters` import on line 4 as well then.

File: __tests__/actions/URLEncode.spec.ts [hunk 2]
```diff
@@ -32,4 +43,20 @@ describe('URLEncode function', () => {
     expect(actual).toEqual(expected);
   });
 
+  it('inverts a URLEncode action', () => {
+    if (URLEncode.invert) {
+      const action = {
+        WFWorkflowActionIdentifier: 'is.workflow.actions.urlencode' as WFWorkflowActionIdentifier,
+        WFWorkflowActionParameters: {
+          WFEncodeMode: '' as WFWorkflowActionParameters['WFEncodeMode'],
```
Comment (joshfarrant) [inline __tests__/actions/URLEncode.spec.ts]: Can this be please be changed to:

```ts
WFEncodeMode: '' as WFEncodeMode,
```
You'll then also need to make sure `WFEncodeMode` is imported at the top, and you can then remove the `WFWorkflowActionParameters` import again.

**Comment by shekhar108** [comment]
I have committed the required changes. Apologies for the late reply.

**Comment by joshfarrant** [comment]
And apologies for my late reply too! Thank you again for taking the time to add this! 😄

**Pull request merged**